client/extensions/zoninator/components/settings/zone/index.jsx

```diff
@@ -79,27 +100,29 @@ class Zone extends Component {
 				) }
 
 				<ZoneDetailsForm
+					disabled={ ! this.locked() }
```

Comment:
As per my comment on the `locked` method, semantically I think this should either be `disabled={ this.locked() }` or `disabled={ ! this.unlocked() }`.